As we use DEB_CONFIG_INSTALL_DIR ?= /usr/share/kde4/config
and we want kdm configuration files in /etc/kde4/kdm,
Let's hardcode it.

Index: kdebase-workspace/kdm/kfrontend/CMakeLists.txt
===================================================================
--- kdebase-workspace.orig/kdm/kfrontend/CMakeLists.txt	2010-10-27 21:55:03.000000000 +0200
+++ kdebase-workspace/kdm/kfrontend/CMakeLists.txt	2010-12-18 21:00:13.950319340 +0100
@@ -112,5 +112,5 @@
 
 # use 'GENKDMCONF_FLAGS=... make install' to add flags to the config generation (try genkdmconf -h)
 install(CODE "
-exec_program(\"${CMAKE_CURRENT_BINARY_DIR}/genkdmconf\" ARGS --in \\\"\\\$DESTDIR${CONFIG_INSTALL_DIR}/kdm\\\" --no-in-notice --face-src \\\"${CMAKE_CURRENT_SOURCE_DIR}/pics\\\" \\\$GENKDMCONF_FLAGS)
+exec_program(\"${CMAKE_CURRENT_BINARY_DIR}/genkdmconf\" ARGS --no-old --in \\\"\\\$DESTDIR/etc/kde4/kdm\\\" --no-in-notice --face-src \\\"${CMAKE_CURRENT_SOURCE_DIR}/pics\\\" \\\$GENKDMCONF_FLAGS)
 ")
